paimon-spark/paimon-spark-3.4/src/test/scala/org/apache/paimon/spark/sql/PaimonOptimizationTest.scala

```diff
@@ -28,7 +28,7 @@ class PaimonOptimizationTest extends PaimonOptimizationTestBase {
       cteIndex: Int,
       output: Seq[Attribute],
       fieldIndex: Int): NamedExpression = {
-    GetStructField(ScalarSubquery(CTERelationRef(cteIndex, _resolved = true, output)), fieldIndex)
+    GetStructField(ScalarSubquery(CTERelationRef(cteIndex, _resolved = true, output, isStreaming = false)), fieldIndex)
```

Comment:
[SPARK-46062][SQL] Sync the isStreaming flag between CTE definition and reference

https://issues.apache.org/jira/browse/SPARK-46062

Fix Version/s: 3.4.2, 4.0.0, 3.5.1

Reply:
Thanks! We need solve this in future.